### DISCUSSION TITLE
React Swiper Pause Autoplay on Mouse Hover

### DISCUSSION THREAD

**Discussion by burhan333** [nolimits4web/swiper#5537] [Q&A]
Is there anyway i can pause autoplay on mouse hover and start autoplay on mouse out in react swiper

**Comment by Arhell** [comment]
https://swiperjs.com/swiper-api#param-autoplay-pauseOnMouseEnter

**Reply by burhan333** [reply]
I already Tried this I don't know why it is not working.
Here you can see my code 
![image](https://user-images.githubusercontent.com/65041476/157756067-118750bd-4aa9-4d76-b56e-b53eebd980fe.png)

**Reply by maimai123** [reply]
> I already Tried this I don't know why it is not working. Here you can see my code ![image](https://user-images.githubusercontent.com/65041476/157756067-118750bd-4aa9-4d76-b56e-b53eebd980fe.png)

the same with u ..

**Comment by hqxc-sumail** [comment]
it doesnt work for me
